domains/authorization/Authorization.API/API/Authorization/[Features]/GrantConsentToOrganizationCommand.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using API.Data;
+using API.Models;
+using API.Repository;
+using API.ValueObjects;
+using EnergyOrigin.Domain.ValueObjects;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using OrganizationId = API.ValueObjects.OrganizationId;
+
+namespace API.Authorization._Features_;
+
+public class GrantConsentToOrganizationCommandHandler(
+    IOrganizationRepository organizationRepository,
+    IUserRepository userRepository,
+    IOrganizationConsentRepository organizationConsentRepository,
+    IUnitOfWork unitOfWork)
+    : IRequestHandler<GrantConsentToOrganizationCommand>
+{
+    public async Task Handle(GrantConsentToOrganizationCommand toOrganizationCommand, CancellationToken cancellationToken)
```

Comment:
grantConsentToOrgCommand

Reply:
Done, same as other grant command, renamed to simply _command_.